README.md

```diff
@@ -12,18 +14,21 @@ The Username and password for the API are:
 The API will need to be authenticated against using HTTP Basic auth and will return a JSON object that contains some queue data containing a list of customer objects. 
 
 # Requirements: 
--  The app should make a request to this api and display the list of customers name and their ‘expectedTime' and ordered by the expected time closest time first.
+
+-  The app should make a request to this api and display the list of customers (`name` and `expectedTime`), ordered by _earliest_ `expectedTime`.
```

Comment:
I think it's nice to let them see what could be important to put on the UI, from themselves they should think about addding name / position/ etc...